src/components/layout/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import NavBar from "components/layout/navbar";
+
+function Layout() {
+  return <NavBar />;
```

Comment:
Layout component should contain the structure of the page and render other page specific components dynamically.